src/app/account-app/account-app.module.ts

```diff
@@ -0,0 +1,172 @@
+  MatExpansionModule,
+  MatFormFieldModule,
+  MatIconModule,
+  MatInputModule,
+  MatListModule,
+  MatProgressSpinnerModule,
+  MatRadioModule,
+  MatSelectModule,
+  MatSidenavModule,
+  MatTableModule,
+  MatToolbarModule,
+  MatTooltipModule,
+} from '@angular/material';
+
+@NgModule({
+  declarations: [
+    AccountAddonsComponent,
+    AccountAppComponent,
+    AccountComponentsComponent,
+    AccountReceiptComponent,
+    AccountRenewalsComponent,
+    AccountTransactionsComponent,
+    AccountUpgradesComponent,
+    BitpayPaymentDialogComponent,
+    ComponentCardComponent,
+    PaymentMethodComponent,
+    PaypalHandlerComponent,
+    PaypalPaymentDialogComponent,
+    ProductComponent,
+    ShoppingCartComponent,
+    StripePaymentDialogComponent,
+  ],
+  imports: [
+    BrowserModule,
+    BrowserAnimationsModule,
+    FormsModule,
+    MenuModule,
+    MatButtonModule,
+    MatCardModule,
+    MatDatepickerModule,
+    MatCheckboxModule,
+    MatDialogModule,
+    MatExpansionModule,
+    MatFormFieldModule,
+    MatIconModule,
+    MatInputModule,
+    MatListModule,
+    MatProgressSpinnerModule,
+    MatRadioModule,
+    MatSelectModule,
+    MatSidenavModule,
+    MatTableModule,
+    MatToolbarModule,
+    MatTooltipModule,
+    ReactiveFormsModule,
+    RouterModule.forChild([
+      {
+        path: 'account',
+        canActivateChild: [RMMAuthGuardService],
+        children: [
+          {
+            path: '', outlet: 'headertoolbar',
+            component: HeaderToolbarComponent
+          },
+          {
+            path: '',
+            component: AccountAppComponent,
+            children: [
+              {
+                  path: 'components',
+                  component: AccountComponentsComponent,
+              },
+              {
+                  path: 'upgrades',
+                  component: AccountUpgradesComponent,
+              },
+              {
+                  path: 'addons',
+                  component: AccountAddonsComponent,
+              },
+              {
+                  path: 'renewals',
+                  component: AccountRenewalsComponent,
+              },
+              {
+                  path: 'transactions',
+                  component: AccountTransactionsComponent,
+              },
+              {
+                  path: 'receipt/:id',
+                  component: AccountReceiptComponent,
+              },
+              {
+                  path: 'cart',
+                  component: ShoppingCartComponent,
+              },
+              {
+                  path: 'paypal/:action',
+                  component: PaypalHandlerComponent,
+              },
+            ]
+          }
+        ]
+      }
+    ]),
+  ],
+  entryComponents: [
+    BitpayPaymentDialogComponent,
+    PaypalPaymentDialogComponent,
+    StripePaymentDialogComponent,
+  ],
+  providers: [
+    PaymentsService,
+    ScriptLoaderService,
+  ],
+  bootstrap: [AccountAppComponent]
```

Comment:
Why bootstrapping ? are you running this standalone in some cases?

Reply:
Hm, I don't think so. Will remove it.

Reply:
Fixed in 0d0e053, will squash them later.